gtdynamics/jumpingrobot/src/jumping_robot.py

```diff
@@ -181,36 +183,42 @@ def create_robot(params, phase) -> gtd.Robot:
         foot_distance = params["morphology"]["foot_dist"]
 
         # compute link, joint poses
-        link_poses, joint_poses = JumpingRobot.compute_poses(length_list, foot_distance)
-        
+        link_poses, joint_poses = JumpingRobot.compute_poses(
+            length_list, foot_distance)
+
         # cosntruct links
         ground = gtd.Link(0, "ground", 1, np.eye(3), Pose3(), Pose3(), True)
-        shank_r = JumpingRobot.construct_link(
-            1, "shank_r", mass_list[4], length_list[4], link_radius, link_poses["shank_r"])
-        thigh_r = JumpingRobot.construct_link(
-            2, "thigh_r", mass_list[3], length_list[3], link_radius, link_poses["thigh_r"])
-        torso = JumpingRobot.construct_link(
-            3, "torso", mass_list[2], length_list[2], link_radius, link_poses["torso"])
-        thigh_l = JumpingRobot.construct_link(
-            4, "thigh_l", mass_list[1], length_list[1], link_radius, link_poses["thigh_l"])
-        shank_l = JumpingRobot.construct_link(
-            5, "shank_l", mass_list[0], length_list[0], link_radius, link_poses["shank_l"])
+        shank_r = JumpingRobot.construct_link(1, "shank_r", mass_list[4],
+                                              length_list[4], link_radius,
+                                              link_poses["shank_r"])
+        thigh_r = JumpingRobot.construct_link(2, "thigh_r", mass_list[3],
+                                              length_list[3], link_radius,
+                                              link_poses["thigh_r"])
+        torso = JumpingRobot.construct_link(3, "torso", mass_list[2],
+                                            length_list[2], link_radius,
```

Comment:
same comment as for the previous file - it also doesn't look like either of these files actually have any change involving the `.key` so I don't think they belong here.

Reply:
Yeah I thought formatting these files made sense now since the main changes in this PR are quite small. I am surprised these were not formatted earlier. 😕

Reply:
I think I may have used a different format option before when writing these files.